backend/realtime_worker.py

```diff
@@ -23,7 +26,8 @@ def fetch_tweets(db, stream):
         print ('Got tweet.')
         if isTransportOffer(tweet):
             print ('Adding tweet from @{}'.format(tweet['user']['screen_name']))
-            r.db('voyageavecmoi').table('offers').insert(get_tweet_data(tweet)).run(db)
+            tweet_data = retry_call(get_tweet_data, (tweet,), tries=MAX_TRIES)
```

Comment:
Alright, I think that we should wrap the iteration in a `try/catch` and make somehow restart the loop.